handout/features/constants,_and_block_scoped_variables.md

```diff
@@ -41,29 +41,26 @@ console.log(j); // 9
 console.log(k); // undefined
 ```
 
-Despite the introduction of block scoping, functions are still _the_ preferred
-mechanism for dealing with most loops.
+Despite the introduction of block scoping, functions are still the preferred mechanism for dealing with most loops.
 
-`let` works like `var` in the sense that its data is read/write. `let` is also useful when used in a for loop. For example, without let:
+`let` works like `var` in the sense that its data is read/write. `let` is also useful when used in a for loop. For example, without let, the following example would output `5,5,5,5,5`:
 
 ```js
 for(var x=0; x<5; x++) {
   setTimeout(()=>console.log(x), 0)
 }
 ```
 
-Would output `5,5,5,5,5`. However, when using `let` instead of `var`, the value would be scoped in a way that people would expect.
+However, when using `let` instead of `var`, the value would be scoped in a way that people would expect.
 
 ```js
 for(let x=0; x<5; x++) {
   setTimeout(()=>console.log(x), 0)
 }
 ```
 
-
-Alternatively, `const` is read only.  Once `const` has been assigned, the identifier can not be
-re-assigned, the value is [not immutable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const).
-
+Alternatively, `const` is read-only.  Once `const` has been assigned, the identifier cannot be
+reassigned; the value is [not immutable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const).
```

Comment:
Might be a bit clearer to write:

"... the identifier cannot be reassigned; however the value itself is still [mutable](..."

What we're getting at here is that some people mistakenly assume `const` results in value immutability; however in reality it only results in reference immutability.